### DISCUSSION TITLE
200w power fluctuations normal?

### DISCUSSION THREAD

**Discussion by savage7** [Arnold-n/P1P2MQTT#146] [General]
It's my first season with a heat pump and I'm not sure if thats normal behavior. Maybe someone could share his experience.

My heat pump runs the whole day at a relativ constant rate, but there are micro power fluctuations with 200w. 
Are these normal? Is it normal that it runs at such a constant rate? It seams to be very efficient, only consuming about 12kwh a day.

Outside conditions on this way are stable at 4-6°.
My current config:
Overshot 4, Modulation On, I have a Madoka installed, setpoint -1

<img width="1187" alt="Bildschirmfoto 2024-11-28 um 18 05 09" src="https://github.com/user-attachments/assets/04f45ade-ca9c-4684-9fa0-70616fae90a3">
<img width="1165" alt="Bildschirmfoto 2024-11-28 um 18 05 38" src="https://github.com/user-attachments/assets/99349178-9c19-4b66-affd-03613642b71c">

**Comment by Arnold-n** [comment]
This is not really normal, but it is typical, Daikin behavior. Must be something in the control loop for the LWT. If it occurs, it often means that the average LWT stays somewhat above the LWT setpoint, increasing the risk of an overshoot and an off/on cycle, so it is recommended to keep the overshoot setting [9-04] sufficiently high (as done). Often, the modulation stops when the LWT setpoint is temporarily increased by 1C which can be done with the offset value (in WD mode) or with the ABS value (in ABS mode).

**Comment by sday** [comment]
I would expect to see energy demands decline throughout the day like that if heating load decreases. (sunlight, wind, etc) Or if the outdoor unit is in the sun, if you're trying to heat your house, the added temperature to the compressor coils would lead to an increase in efficiency so the dynamic nature of inverter driven compressors would probably request a lower speed, close the EEV a bit and maybe lower indoor fan speed slightly. It seems like at 0800, 0930 it stepped down, then maybe someone turned up the temp, or opened the door at 1100 so it ramped up, resetting the baseline, but then it ramped back down at 1730. I could be totally looney, but that's what I got out of the graph. :)

**Comment by savage7** [comment]
@sday 
I enabled averaging the temperature over 24h. I guess the steps are more are reaction to outside temperature changes. 

@Arnold-n 
I played around and manually changed the setpoint +/- 1C, in some cases it helped in most it didn't. When it "worked", it goes into "slower" fluctuations like this: 
<img width="643" alt="Bildschirmfoto 2024-12-10 um 15 36 31" src="https://github.com/user-attachments/assets/ea0522c5-7549-4cd6-a3b7-0cef3d85abe8">

Maybe the 24h average causes this. I'll try to disable it again. It solved my frequent defrost cycle problem.

**Reply by Arnold-n** [reply]
Thanks - slower fluctuations are related to the fact that is has a limited number of power states. I am pretty sure 24h is not the cause and is beneficial to prevent the fast-changing pattern. In my experience it starts when LWT is well above setpoint and going down quickly; which happens when the systems starts and overshoots, or when the setpoint goes down too fast.

**Comment by savage7** [comment]
I've monitored these fluctuations over the past months, they seam to be normal for this unit. At least I found no setting that would change this behaviour.
